curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/guided-project-calculate-and-print-student-grades.md

```diff
@@ -9,14 +9,16 @@ dashedName: guided-project-calculate-and-print-student-grades
 
 This challenge will be partially completed on Microsoft's learn platform. Follow these instructions to complete the challenge:
 
-1. Go to <a href="https://learn.microsoft.com/en-us/training/modules/guided-project-calculate-print-student-grades/" target="_blank">https://learn.microsoft.com/en-us/training/modules/guided-project-calculate-print-student-grades/</a> and complete all the tasks for the "Guided Project - Calculate and Print Student Grades" module. This is required to earn the "Write Your First Code Using C#" trophy.
+1. Go to <a href="https://learn.microsoft.com/training/modules/guided-project-calculate-print-student-grades/" target="_blank">https://learn.microsoft.com/training/modules/guided-project-calculate-print-student-grades/</a> and complete all the tasks for the "Guided Project - Calculate and Print Student Grades" module. This is required to earn the "Write Your First Code Using C#" trophy.
 1. When you are finished, come back and correctly answer the question below.
 
 # --question--
 
 ## --text--
 
-What is wrong with the following code? `int sophiaSum; Console.WriteLine("Sophia: " + sophiaSum);`
+What is wrong with the following code?
+
+`int sophiaSum; Console.WriteLine("Sophia: " + sophiaSum);`
```

Comment:
````suggestion
```c++
int sophiaSum; Console.WriteLine("Sophia: " + sophiaSum);
```
````

If these are not going to be in-line, should we use proper code blocks?

Reply:
You're probably right - but I didn't really like the look of it like that. The purpose of the adjustments on these was just to create space to make it more readable. If we used codeblocks, there would only be one or two challenges that would use them, so I thought this would look more consistent. I would prefer to just leave them.